src/server/rest.cpp

```diff
@@ -79,6 +95,11 @@ namespace ouroboros
 
 		std::string result;
 		result.assign(match[0].ptr, match[0].len);
+		
+		if(result != "server"){
+			result.insert(0, "server-");
+		}
+
```

Comment:
Yeah, this was the issue that I was seeing earlier. We can either disallow direct access to the root group (I'm not in favor of this approach), or reserve a word explicitly (something like _root). What this would allow for, then, is for users to access data either by using:
`/groups/_root-group_a/fields/item_a`
or
`/groups/group_a/fields/item_a`